src/libraries/Microsoft.PowerFx.Interpreter/UDFTexlFunction.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx
+{
+    public sealed partial class RecalcEngine
```

Comment:
Do these all need to be inner classes of RecalcEngine?

Reply:
Fixed